content/guides/building/get-source-git.md

```diff
@@ -48,64 +48,75 @@ to the last 10 commits, for example.
 
-Before making your first commit on a new system, be <strong>sure</strong> to configure Git. These global settings are stored in your git configuration directory (`~/.git/` or for Haiku: `~config/settings/git/`) and will be appended to <strong>each</strong> commit as your personal information.
+Using `--global` will your personal information in your Git configuration directory (`~/.git/` or for Haiku: `~config/settings/git/`), which will be included in <strong>every</strong> commit.
 
 ```sh
 git config --global user.name "John Doe"
-git config --global user.email "john.doe@developers.com"
+git config --global user.email "john.doe@example.com"
 ```
 
-On macOS, you must set the following option in order to avoid problems with the unicode representation of filenames:
+On macOS, you must set the following option in order to avoid problems with the Unicode representation of filenames:
 
 ```sh
 git config core.precomposeunicode true
 ```
 
 <h4>Setup an account on Gerrit</h4>
 
-<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. You currently need a <a href="https://github.com">Github</a> account for logging in.</p>
+<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. For the time being, using a <a href="https://github.com">GitHub account</a> is necessary.</p>
 
 <div class="alert alert-warning">
-Make sure your ssh key is generated with ed25519. RSA keys will not work with Gerrit.
+Make sure your SSH key is generated using <code class="code">ed25519</code>. RSA keys will <b>not</b> work with Gerrit.
 </div>
-<p>Upload your SSH public key in <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
+<p>Upload your SSH public key in the <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH Keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
 <p>If the e-mail address used in your commits does not match the one in your github account, you will need to add and verify it in <a href="https://review.haiku-os.org/settings/#EmailAddresses">Gerrit E-Mail address settings</a>.</p>
 
-<h4>Build Tools:</h4>
+<div class="alert alert-warning">
+You may need to replace the <code class="varname">$USER</code> parameter in the following commands.
+</div>
+<p>This can happen if the username on your Gerrit account does not match the username that you use to log in to your computer. This would be the case in Haiku itself, as the default username is <code class="code">user</code>. You can make your life easier by <a href="https://linux.die.net/man/5/ssh_config">modifying your SSH configuration</a>.</p>
+<p><code class="varname">$USER</code> is an environment variable that is specific to Unix-like operating systems (e.g. Haiku, Linux, FreeBSD, etc.). If your system does not fall under that category, you will probably have to replace it.</p>
```

Comment:
Should this be removed? The platforms that we support are explicitly stated here: https://github.com/haiku/haiku/blob/07f11ccca916aa828439b25d51713f1cf610b79a/ReadMe.Compiling.md#building-haiku, however, the document does not mention this as it is out of scope.

However, this is the first thing people should supposedly read before they compile Haiku. How do I work around that?

Reply:
Really this should be all moved to https://www.haiku-os.org/docs/develop/ where we can have a book-like thing, with pages you have to read in order. The website does not provide that, I think, unless we start manually adding a kind of "next/previous page" link somewhere?

Maybe add yet another warning to this page linking to the list of supported platforms.

Reply:
Let's just remove the warning for the time being and hope that the user won't use Windows?

Reply:
P.S. That's a story for another day, but it's a bit hard to move the documents, as they all have to be moved at once and I am not sure what's the standard practice on introducing redirects. (I know where to put them, but has this been done before? It's why I never got around to it.

Reply:
we had one confused user try this under WSL2, and it fails only much later. So it would be nice to keep the warning.

Reply:
> we had one confused user try this under WSL2, and it fails only much later.

good god

Reply:
I think the simplest way to proceed here is not try to rely on $USER being already set to the right thing.

So let's set it manually:

    set GERRIT_USER=your gerrit username
    git clone $GERRIT_USER@...

And maybe add a screenshot showing where to find the username in Gerrit